.eslintrc.json

```diff
@@ -26,6 +26,10 @@
     "no-underscore-dangle": "off",
     "no-return-await": "off",
     "@typescript-eslint/return-await": "off",
-    "object-curly-spacing": "error"
+    "object-curly-spacing": "error",
+
+    // airbnb terrible defaults
+    "no-restricted-syntax": ["error", "ForInStatement", "LabeledStatement", "WithStatement"],
+    "no-continue": "off"
```

Comment:
Yeah no. There's nothing wrong with `continue`.

Reply:
I think it's just the airbnb settings we base things on. I've cargo-culted the sam, but I've started to think their rules might be a bit too outdated by now.